lib/BackgroundJob/SyncJob.php

```diff
@@ -79,6 +84,12 @@ protected function run($argument) {
 			return;
 		}
 
+		if (($this->userConfig->getValueInt($account->getUserId(), Application::APP_ID, 'ui-heartbeat') - $this->time->getTime()) > 900) {
+			$this->logger->debug('Detected user activity, skipping background sync job');
+			$this->setInterval(900);
```

Comment:
Does setting the interval of an already running job have any consequence?

Reply:
Yes, it is saved by the Job base class when it schedules the next execution.